src/commands/core/asset/create.ts

```
@@ -22,206 +22,341 @@
+    This wizard will guide you through the process of creating a new asset.                
+                
+--------------------------------`
+      )
 
-      fs.writeFileSync(flags.json, JSON.stringify(jsonFile, null, 2))
+      const wizard = await createAssetPrompt()
+      const pluginData = wizard.plugins
 
-      const jsonSpinner = ora('Uploading JSON...').start()
+      // Create new metadata from wizard answers
+      let imageUri: string | undefined
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'imageUri' is only assigned here.


(lint/style/useConst)